pkg/cmd/factory/factory_impl.go

```diff
@@ -19,6 +19,8 @@ import (
 	"time"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/kv"
+	cdckv "github.com/pingcap/tiflow/cdc/kv"
```

Comment:
in some file, `/tidb/kv` was renamed to `tidbkv`.
I would prefer to follow that convention. `kv` for `/cdc/kv`